src/main/rx/ghst.c

```diff
@@ -252,6 +269,9 @@ static bool ghstProcessFrame(const rxRuntimeState_t *rxRuntimeState)
                 case GHST_UL_RC_CHANS_HS4_5TO8:     startIdx = 4;  break;
                 case GHST_UL_RC_CHANS_HS4_9TO12:    startIdx = 8;  break;
                 case GHST_UL_RC_CHANS_HS4_13TO16:   startIdx = 12; break;
+                default:
+                    DEBUG_SET(DEBUG_GHST, DEBUG_GHST_UNKNOWN_FRAMES, ++unknownFrameCount);
```

Comment:
I'm not sure how much value the "unknown" frame count is. Doesn't that somewhat defeat the purpose of the previous changes to silently ignore unsupported frame types? As you add new types to the receiver protocol in the future then this debug may start to confuse users into thinking there's a problem.

Reply:
With the current development going on in the protocol this is required.